UltiSnips/markdown.snippets

```diff
@@ -50,6 +50,23 @@ snippet spar "Paragraph" b
 $0
 endsnippet
 
+###################
+# Text formatting #
+###################
+
+snippet * "cursive"
+*${1:${VISUAL}}*$0
+endsnippet
+
+snippet ** "bold"
+**${1:${VISUAL}}**$0
+endsnippet
+
+snippet *** "bold"
```

Comment:
Is this text right?

Reply:
No, this was a copy & paste error. I corrected it in b8f54d395e2f0b20b0ae12e3e21dddad2a0b3fda.